docs/contributor/data.md

```
@@ -443,6 +443,56 @@ spec:
+  validation:
+    performance:
+      checks:
+        - nccl-all-reduce-bw
+      constraints:
+        - name: nccl-all-reduce-bw
+          value: ">= 720"
+```
+
+When a query specifies `{service: eks, accelerator: gb200, intent: training}`, the resolver returns both the service-specific inheritance chain (`gb200-eks-training` → `eks-training` → `base`) and the wildcard overlay (`gb200-any-training`) independently. Both contribute to the merged result:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)